### PR TITLE
PP-136/smart wallet transfer

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

Removed the process where native cryptocurrency balance is transfer back to the owner EOA

## Why

It would be more consistent (simpler, in UX terms) to leave the native cryptocurrency balance in the SmartWallet instead of transferring it to the owner EOA after each execution.

## Refs

- [Issue](https://github.com/rsksmart/rif-relay/issues/179)

### CODE REVIEW

**Review by antomor — APPROVED** [review]
LVGTM!

**Comment by antomor** [comment]
@franciscotobar Thank you for this PR. Before merging it, we may need to change the [dapp](https://github.com/rsksmart/relaying-services-sdk-dapp) to reflect these changes.

File: contract-addresses.json [hunk 1]
```diff
@@ -1,30 +1 @@
...
...:"0x25b03F90DD4D20Ba500b1950e3E3044ebC49E5fD","customSmartWalletFactory":"0x425A3d21eB298480b45d5f6D9b20cAcc27660E01","customSmartWalletDeployVerifier":"0x24445B6454cA52f6d8B2716952dA9D1Fc30B4344","customSmartWalletRelayVerifier":"0x9c29fEE0317178c69c411aeD358Ffb17E4EbFe29","sampleRecipient":"0x22167CE0c7646C49049a6Ae9F877a14310d24B4E","testToken":"0xF5859303f76596dD558B438b18d0Ce0e1660F3ea"},"33":{"penalizer":"0x7562af45f4D0F1C3a3e09ba62ABd61dec68aB925","relayHub":"0x4Bc162B5ed6967E8c7F38d46a6d56ec122ffca43","smartWallet":"0xD1003d10f6E426d1B1c82CEbe5E1cb4128589deB","smartWalletFactory":"0xfb633E977B84ffF04eEc12C3c9785caD3846d9fD","smartWalletDeployVerifier":"0xB4c021f13840880b04C985dc5ef07bb8A484dF59","smartWalletRelayVerifier":"0x829Bfae4c492b737BE00B635076daB2BDFc46c55","customSmartWallet":"0x267D1cABA390C21df371aD02fDeF2dbc966a015C","customSmartWalletFactory":"0x729f67cB6478C30F36b772bc5A70aB568e9DC21e","customSmartWalletDeployVerifier":"0x1bF847Ff9800F3E6C03Ea4780e4F4249982c851C","customSmartWalletRelayVerifier":"0x0ae32078Fd1A3563cdC43672D1D78cDeB9fC177b","sampleRecipient":"0xFAF0d39D89D41A6A1E6D939cD55126a46a1E75B1","testToken":"0x86c406E582A360A1BfE363e28988Ae96780bEEe3"}}
```
[the remark this answers is not in the document]
Comment (jurajpiar) [reply contract-addresses.json:1]: thought this file was meant to be generated in deployment?

**Comment by antomor** [comment]
We decide to [implement a new SmartWallet](https://github.com/rsksmart/rif-relay-contracts/pull/117) instead of changing the existing one.

**Pull request closed without merging**